app/va/va_profile/va_profile_client.py

```diff
@@ -124,6 +187,60 @@ def get_telephone(
         self.statsd_client.incr('clients.va-profile.get-telephone.failure')
         self._raise_no_contact_info_exception(self.PHONE_BIO_TYPE, va_profile_id, response.get(self.TX_AUDIT_ID))
 
+    def get_is_communication_allowed_api_v3(
+        self,
+        recipient_id: 'RecipientIdentifier',
+        communication_item_id: str,
+        notification_id: str,
+        notification_type: str,
+    ) -> bool:
+        from app.models import VA_NOTIFY_TO_VA_PROFILE_NOTIFICATION_TYPES
```

Comment:
With very few exceptions, imports should not happen inside functions. The use case for this dictionary is only in this file. Please remove the import and declare it in this file/class. It is specific to VA Profile and I'm not sure why it's in models (no need to change the other code outside your PR). We will remove it from app.models when we clean up the old `get_is_communication_allowed_api` method.